environments/test-west-region2/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+   required_version = ">= 1.2.0"
+  required_providers {
```

Comment:
Running 'terraform fmt -recursive -diff' from the root directory before committing is a good habit to keep indentation consistent

Reply:
My bad! Should have done it, will fix it up!